src/interfaces/events-query.ts

```diff
@@ -39,7 +36,7 @@ export class EventsQuery extends AbstractMessage<EventsQueryMessage>{
     const descriptor: EventsQueryDescriptor = {
       interface        : DwnInterfaceName.Events,
       method           : DwnMethodName.Query,
-      filters          : options.filters ? Events.normalizeFilters(options.filters) : undefined,
+      filters          : options.filters ? Events.normalizeFilters(options.filters) : [],
```

Comment:
Unrelated to this PR, but didn't like the duality between an undefined filter and empty array, so made filters required and allowed an empty array in the message descriptor.